source/FreeRTOS_IP_Utils.c

```diff
@@ -87,6 +87,10 @@
     static UBaseType_t uxLastMinQueueSpace = 0;
 #endif
 
+/* IPv4 multi-cast addresses range from 224.0.0.0.0 to 240.0.0.0. */
```

Comment:
Why moved to common code? If we adding to common code we should think about renaming the names, so it is not confusing.

Reply:
Is it used for comparison?